### DISCUSSION TITLE
What is the last version that still works on Python 2

### DISCUSSION THREAD

**Discussion by janicijevic82** [AzureAD/microsoft-authentication-library-for-python#465] [Q&A] [answered]
I am currently using version 1.12.0 and it works with Python 2 without problem. I wanna update it to the 1.17.0, but as a prereq said that it has be running on Python 3.6+. Can anyone told me what is the highest version that still works on Python 2.

Thanks 
Nikola

**Comment by rayluo** [comment, marked as answer]
MSAL for Python currently still supports Python 2. You can get this info from the "Programming Language" section at the left sidebar on the PyPI page. https://pypi.org/project/msal/

So, you can use the latest version of MSAL for Python, v1.17.

**Reply by janicijevic82** [reply]
Hi Ray, 
Thanks one more time for both answer and for help.

Nikola
